data/fixtures/recorded/languages/clojure/clearEveryItem.yml

```diff
@@ -27,11 +27,10 @@ finalState:
 
     {
         ,
-        ;; hello
```

Comment:
NTS: This is an example of the new code performing worse, but we understand why. This is because textual scope defines the item as:
```
;; hello
:baz "whatever",
```

In this case the language does identify that the comment is not part of the item, but we are preferring the textual scope, and as above, we no longer blind take the language server's view.

This will affect every language will you have an inline comment next to an item, we just don't have many tests in our more common languages that show that. There's an issue for improving the handling of comments around items. (Might be https://github.com/cursorless-dev/cursorless/issues/1770?). It's a complex change.

Reply:
NB: the newer code uses more queries and predicates and less imperative TypeScript, so it's also something that would have been easier to monkey patch in the older code, although arguably, for a worse long term outcome